Interfaces/API/DiracAdmin.py

```diff
@@ -136,7 +136,7 @@ def getSiteMask(self, printOutput=False, status='Active'):
     return result
 
   #############################################################################
-  def getBannedSites(self, gridType=[], printOutput=False):
+  def getBannedSites(self, printOutput=False):
```

Comment:
That's a change in the external API, if one wanted to be really strict, we should change major version number

Reply:
It's anyway unused...